packages/db-react/src/use-root-loader-data.tsx

```diff
@@ -0,0 +1,6 @@
+import { useRouteLoaderData } from 'react-router'
+import type { DbLoaderData } from './db-loader.tsx'
+
+export function useRootLoaderData() {
```

Comment:
After using `useRouteLoaderData<DbLoaderData>('root')` for the 4th time I figured now it's time to create a declarative re-usable function instead of imperatively doing this each time around.